src/alter_table.cr

```diff
@@ -199,15 +199,15 @@ module Cql
     # sql = to_sql(visitor)
     # ```
     def to_sql(visitor : Expression::Visitor)
-      String::Builder.build do |sb|
+      String.build do |string|
```

Comment:
Replaced direct usage of `String::Builder.build` with `String.build` as recommended in [the documentation](https://crystal-lang.org/api/1.13.3/String/Builder.html). Additionally, changed the variable name from `sb` to `string` to address the complaint from Ameba.